simplivity/resources/backups.py

```diff
@@ -141,6 +142,38 @@ def delete_multiple_backups(self, backups, timeout=-1):
 
         self._client.do_post(method_url, data, timeout, None)
 
+    def set_retention(self, backups, retention, force=False, cluster_group=None, timeout=-1):
+        """Sets the retention time for the specified list of backups.
+
+        Args:
+          backups: The list of backup objects that you want to set the retention time for.
+          retention: The number of minutes to keep backups.
+          force: An indicator to force a retention time modification even if this action results in deleting backups.
+            Valid Values:
+              True: Sets the new retention time for the specified list of backups. This action may delete of one or more
+                    backups.
+              False: Does not make the requested retention time modification if this results in deleting one or more
+                     backups. This operation returns a list of backups that the requested modification deletes. If the
+                     requested modification does not delete the backups, the retention time modification occurs.
+          cluster_group: Object/name of the cluster group.
+          timeout: Time out for the request in seconds.
+
+        Returns:
+          list: List of backup objects.
+        """
+        method_url = "{}/set_retention".format(URL)
+        backup_ids = [backup.data["id"] for backup in backups]
+        data = {"backup_id": backup_ids, "retention": retention, "force": force}
+        if cluster_group:
```

Comment:
It's good to have a line break before 'if' and before 'return' statements to make the code more readable.

Reply:
Added the line break as suggested